2d_motion/convert_2d.m

```diff
@@ -12,7 +12,7 @@
 % Author: Jason Balke | Project: CAST | Date: 10/20/20
 %----------------------------------------------------------------------------------------%
 % Get normal vector to collision plane and location of collision
-planeUnitVec = refOrbit(1,1:3)/norm(refOrbit(1,1:3));
+planeNormalVec = cross(refOrbit(1,4:6),conjOrbit(1,4:6));
```

Comment:
normal vec should be (refOrbit(2, 4:6) - conjOrbit(2, 4:6)) as the relative velocity vector